ingress/controllers/nginx/README.md

```diff
@@ -260,239 +258,6 @@ In case of an error in a request the body of the response is obtained from the `
 Using this two headers is possible to use a custom backend service like [this one](https://github.com/aledbf/contrib/tree/nginx-debug-server/Ingress/images/nginx-error-server) that inspect each request and returns a custom error page with the format expected by the client. Please check the example [custom-errors](examples/custom-errors/README.md)
 
 
-### Annotations
```

Comment:
@databus23 this is being added here https://github.com/kubernetes/contrib/pull/1130